galleries/plot_types/basic/README.rst

```diff
@@ -1,6 +1,6 @@
 .. _basic_plots:
 
-Basic
------
+Tabular and linear data
```

Comment:
“paired data”?  as in (x, y) pairs?

Reply:
@story645, the word "tabular" appears exactly 0 times in the docs.  I don't think we should ever use it, because we don't deal with tabular data, and certainly we should not use it in such a prominent place.  

"Basic" comes from the cheatsheet.  I'm not wedded to it, but it is not meant to mean anything other than the plots you'd expect to see in any plotting package.  

> is that most of these plots are implicitly multivariate

Many of these plots can definitely be used for multivariate analysis, but I think the use cases are significantly more general.  Very often we have dependent variable (y) plotted versus independent (x or t), which I wouldn't call "multivariate".

Reply:
Sorry, "don't deal directly with tabular data" is more accurate.  However, the point is that it is a non-existent term in our docs and should be avoided here.  

> It is though since x or t does not necessarily have to be the same type as y - like time vs snow, 

Multivarate statistics is snow vs time, rain vs time, solar incidence versus time, and then all of them versus each other.  Snow versus time by itself is univariate.  https://en.wikipedia.org/wiki/Multivariate_statistics

> whereas in an array it has to be [time, time....time] or [(direction, magnitude), (direction, magnitude)], etc...

When we do `ax.plot(t, y)` the objects `t` and `y` are arrays.  The doctoring for `plot` reads: 

```
x, yarray-like or scalar